tests/testthat/test-map_df.R

```diff
@@ -23,6 +23,48 @@ test_that("map_df works as expected", {
 
   expect_equal(mapping, out)
   expect_equal(sort(unique(unname(unlist(mapping)))), sort(rownames(ref_df)))
+  
+  trt_dt <- dt <- data.table(
```

Comment:
Please, ad short descriptio for tested scenario - for future us 😉

Reply:
ok, done